### ISSUE TITLE
sudden unrelated flake8 fails

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#3220]
https://travis-ci.org/borgbackup/borg/jobs/292737377

note that these files have not been changed in that PR.
so it might be due to a change in flake8, maybe they have new stuff to warn about since recently.

can be "fixed" in 2 ways:

- ignore / suppress warning (see setup.cfg)
- fix the stuff flake8 complains about (be careful)

**Comment by ThomasWaldmann** [comment]
```
src/borg/cache.py:812:13: E722 do not use bare except'
src/borg/testsuite/crypto.py:229:9: E741 ambiguous variable name 'l'
```

**Comment by ThomasWaldmann** [comment]
"fixed" by #3231.
